docs/DeveloperGuide.md

```diff
@@ -886,6 +886,78 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
 - `delete 100000`.
 - `delete1`.
 
+### Rescheduling a meeting
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The following test
+    cases are based on the assumption that the current system datetime is before 20th June 2024.
+    If the current system datetime is after 20th June 2024, the year of the input datetimes must be set to a future year.
+</div>
+
+#### Rescheduling a meeting successfully
+
+**Prerequisites:**
+1. List all persons using the `list` command. At least 5 persons are in the list.
+2. First three person in the displayed list has scheduled meetings.
+
+**Example test case:** `reschedule 1 s/01-12-2024T09:30 e/01-12-2024T12:30`.
+
+**Expected Result:**
+1. The meeting details of the specified person is updated with the provided rescheduled meeting date and time.
+2. *Command Result Box* outputs a successful execution message with the rescheduled meeting information.
+4. *Command Box* is cleared.
+
+**Alternative Test Cases:**
+- `reschedule 2 s/20-06-2024T09:30 e/20-06-2024T12:30`.
+- `reschedule 3 s/15-09-2024T10:00 e/15-09-2024T11:00`.
+
+#### Rescheduling a meeting today to another day successfully 
+
+**Prerequisites:**
+1. List all persons using the `list` command. At least 5 persons are in the list.
+2. The first person in the list has a scheduled meeting today.
+
```

Comment:
I missed this out but I think it would be better to state to include this in the prerequisite:
```suggestion
3. **The scheduled meeting date of this test case must be the same as system's current date**

```